lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -155,7 +155,7 @@ public void Dispose()
             CleanUp();
         }
 
-        protected void ValidateSuccessForLocalRuntime(int numberOfContextsToClose, int numberOfTasksToFail = 0, int numberOfEvaluatorsToFail = 0, string testFolder = DefaultRuntimeFolder, int retryCount = 60)
+        protected void ValidateSuccessForLocalRuntime(int? numberOfContextsToClose, int numberOfTasksToFail = 0, int numberOfEvaluatorsToFail = 0, string testFolder = DefaultRuntimeFolder, int retryCount = 60, string expectedCancellationMessage = null)
```

Comment:
For example, in your test, 
            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder);
            var expectedCancellationMessageCount = GetMessageCount(lines, expectedCancellationMessage );
            Assert.True(expectedCancellationMessageCount > 1 );